python_modules/dagster/dagster/_cli/api.py

```diff
@@ -87,11 +88,11 @@ def send_to_buffer(event):
 
 
 def _should_start_metrics_thread(dagster_run: DagsterRun) -> bool:
-    return get_boolean_tag_value(dagster_run.tags.get("dagster/run_metrics"))
+    return any(get_boolean_tag_value(dagster_run.tags.get(tag)) for tag in RUN_METRIC_TAGS)
```

Comment:
this includes both hidden and non-hidden tag i assume

Reply:
Yes, the list could be modified in one place if we want to retire the public tag completely.